tests/depedency/r-version.R

```diff
@@ -0,0 +1,26 @@
+require(stringi)
+
+print_depends <- function(package, level = 0) {
```

Comment:
Are there tests for this?

Reply:
This is a utility for us to to through all the R version requirement. We can remove and save somewhere else.